classes/Store.php

```diff
@@ -59,7 +59,7 @@ class StoreCore extends ObjectModel
     /** @var float Longitude */
     public $longitude;
 
-    /** @var string Store hours (PHP serialized) */
+    /** @var string|array Store hours (PHP serialized) */
```

Comment:
How it could be an array?

Reply:
In `AdminImportController`, the store can be defined as array.